compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -420,9 +420,6 @@ error.type.cast.not.yet.supported.for.type=\
 error.equality.not.yet.supported.for.type=\
   equality not yet supported for type ''{0}''
 
-error.binding.pattern.not.yet.supported.for.type=\
```

Comment:
This is not the one relevant for BINDING_PATTERN_NOT_YET_SUPPORTED_IN_MODULE_VAR_DECL right? Seems like both can be removed though.

Reply:
Will remove both. thanx